server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JmapGuiceProbe.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.james.core.Username;
```

Comment:
` JAMES-XXXX Adapt MailboxChangesMethodContract for stability against distributed environment`

I'm picky but can you edit the commit message to explain what adaptation you did? Why it is necessary? Why it do not lead to errors?